packages/databricks-vscode/src/ui/bundle-variables/LookupVariableTreeNode.ts

```diff
@@ -0,0 +1,37 @@
+import {TreeItemCollapsibleState} from "vscode";
+import {BundleVariable} from "../../bundle/models/BundleVariableModel";
+import {BundleVariableTreeItem, BundleVariableTreeNode} from "./types";
+import {TreeItemTreeNode} from "../TreeItemTreeNode";
+
+export class LookupVariableTreeNode implements BundleVariableTreeNode {
+    constructor(
+        public readonly key: string,
+        public readonly value: BundleVariable,
+        public readonly parent?: BundleVariableTreeNode
+    ) {}
+
+    getTreeItem(): BundleVariableTreeItem {
+        const lookup = this.value.lookup as any;
+        const lookupValue = Object.values(lookup).at(0) as string | undefined;
```

Comment:
Why are we not checking the first non undefined value instead? Similar to how we check the key in the `getChildren`.